src/ApiGateway/WebsocketClientStatus.php

```diff
@@ -0,0 +1,56 @@
+<?php declare(strict_types=1);
+
+namespace Bref\ApiGateway;
+
+class WebsocketClientStatus
+{
+    /** @var string */
+    private $sourceIp;
+
+    /** @var string */
+    private $userAgent;
+
+    /** @var string */
+    private $connectedAt;
+
+    /** @var string */
+    private $lastActiveAt;
```

Comment:
Should these 2 variables be `DateTimeImmutable` instances?

Reply:
@mnapoli Right. That would make sense. I'll change this.